Makefile.d/tools.mk

```
@@ -77,7 +77,8 @@ $(BINDIR)/kubectl:
 	chmod a+x $(BINDIR)/kubectl
 else
 $(BINDIR)/kubectl:
-	curl -L "https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl" -o $(BINDIR)/kubectl
+	curl -LO "https://dl.k8s.io/release/$(shell curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+	install -o root -g root -m 0755 kubectl $(BINDIR)/kubectl
```

I think user and permission is up to the user environment, how about using $USER for user and group definition
```suggestion
	install -o $USER -g $(id -g $USER) -m 0755 kubectl $(BINDIR)/kubectl
```

---

And you can use it like
```
make USER=root kubectl/install
```

---

Probably this is enough?

https://github.com/vdaas/vald/pull/2032/commits/1355abde2321898e64eeec07c6f402448d498c48

---

Looks enough.